modules.d/95hwdb/module-setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+check() {
+    return 255
+}
+
+# called by dracut
+install() {
+    local hwdb_bin
+
+    # systemd-hwdb ships the file in /etc, with /usr/lib as an alternative.
+    # The alternative location is preferred, as we can consider it being user
+    # configuration.
```

Comment:
This comment is confusing. Do you mean we prefer the `/usr/lib` location or the `/etc` location?

Reply:
Yes, that's what I meant.

From man page:
```
       --usr
           Generate in /usr/lib/udev instead of /etc/udev.
```

Reply:
Usually, it should be only in one place. It's unclear which one, but copying both would be even more confusing. Although copying what `systemd-hwdb query ` uses might be better...